phpBB/phpbb/db/migration/data/v33x/remove_non_existant_assigned_roles.php

```diff
@@ -0,0 +1,72 @@
+<?php
+/**
+*
+* This file is part of the phpBB Forum Software package.
+*
+* @copyright (c) phpBB Limited <https://www.phpbb.com>
+* @license GNU General Public License, version 2 (GPL-2.0)
+*
+* For full copyright and license information, please see
+* the docs/CREDITS.txt file.
+*
+*/
+
+namespace phpbb\db\migration\data\v33x;
+
+class remove_non_existant_assigned_roles extends \phpbb\db\migration\migration
```

Comment:
existent should really be existent. But perhaps an even simpler naming `remove_orphaned_roles`?

Reply:
I vote for `remove_orphaned_roles`